messaging/src/main/java/org/axonframework/messaging/responsetypes/ResponseType.java

```diff
@@ -38,20 +38,43 @@
  */
 public interface ResponseType<R> extends Serializable {
 
+    int NO_MATCH = 0;
+    int MATCHES_SINGLE = 1;
```

Comment:
I would rename this to `SINGLE_MATCH`, since that's in line with `NO_MATCH`.

Reply:
Renamed